recipes/idds-workflow/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "idds-workflow" %}
+{% set version = "0.5.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: cfb19c04c716beabb9c330edf976eb5a8e23c4365094c46f21f10b2875f31512
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - idds-common ==0.5.2
+    - pip
+    - python
+  run:
+    - idds-common ==0.5.2
+    - pip
+    - python
```

Comment:
A few comments here:

1. The host section should only contain packages needed at install-time. My guess is that you only need python and pip there. 

2. The run section should contain only stuff needed at run-time. In this case, you should remove pip like we did last time.

3. If the version of idds-workflow is supposed to match idds-common always, you should use jinja2 like this

```
- idds-common =={{ version }}
```